flutter/android/src/main/kotlin/io/sentry/flutter/SentryFlutterPlugin.kt

```diff
@@ -110,13 +111,11 @@ class SentryFlutterPlugin : FlutterPlugin, MethodCallHandler {
       options.setBeforeSend { event, _ ->
         setEventOriginTag(event)
         addPackages(event, options.sdkVersion)
-        removeThreadsIfNotAndroid(event)
         // TODO: merge debug images from Native
```

Comment:
Do we already have an issue for this?

Reply:
no, and I don't think it's relevant anymore, could be removed, iirc this is done by `loadImageList` method

Reply:
```suggestion
```